Add license file

Monty opened the pull request:
Pull this to add a license file with you as the copyright holder. There shouldn't be any other changes.

Thanks again!

Monty committed:
Create LICENSE

Added an MIT license with copyright by the original developer

Monty committed:
Merge branch 'ozboware:main' into main

ozboware commented:
I've updated the game to include scores, multiple level and difficulty choice and a start up menu which includes access to high scores, best streaks, instructions and difficulty/level changes. I've also included a few animations and modals for the notifications as well as added in some responsive design. It can also be demoed in full online at [wordled.online](https://wordled.online)

**screenshots**
![difficult mode](https://user-images.githubusercontent.com/95859352/152355873-fe134a9e-b62a-4206-860d-5bd2baca2fc7.png)
![difficulty select](https://user-images.githubusercontent.com/95859352/152355874-80be2392-0309-4c34-bd80-debebf7bea01.png)
![game board](https://user-images.githubusercontent.com/95859352/152355876-f30025fa-3483-4f82-8bd3-53ff5aa9113b.png)
![give up screen](https://user-images.githubusercontent.com/95859352/152355877-eb173143-2f66-4819-b392-4c9132927ce6.png)
![help game](https://user-images.githubusercontent.com/95859352/152355879-c1be37f9-b531-429e-8760-eb64a4a5b067.png)
![help options](https://user-images.githubusercontent.com/95859352/152355881-c7b4d12c-16d3-401c-802b-f312a6de7cc4.png)
![in game difficulty select](https://user-images.githubusercontent.com/95859352/152355884-9cd21401-e690-4264-bf7b-6b385613a02e.png)
![in game level select](https://user-images.githubusercontent.com/95859352/152355887-16100666-548e-4a43-bcf0-80500f9df0ee.png)
![level select](https://user-images.githubusercontent.com/95859352/152355889-555833d1-b37f-44f6-8c28-8610b79c67aa.png)
![lost screen](https://user-images.githubusercontent.com/95859352/152355892-4dd37e96-eb8f-4c55-ae18-7f57e0d97259.png)
![menu](https://user-images.githubusercontent.com/95859352/152355893-127166c2-3edd-421e-a8b0-5fe759f09c03.png)
![score easy](https://user-images.githubusercontent.com/95859352/152355896-67098033-52b7-4ad9-ba46-29e02e91e06d.png)
![streak easy](https://user-images.githubusercontent.com/95859352/152355897-e6dd2bb0-9b08-45f7-9542-26ad2f23c478.png)
![win screen](https://user-images.githubusercontent.com/95859352/152355899-a32375af-4f28-4bdd-9c0a-b8488d03e70c.png)

Monty committed:
Merge branch 'ozboware:main' into main

Monty committed:
Fix typos in README

Monty commented:
Impressive! 

I didn’t expect to see so many improvements. Maybe I need to learn Javascript after all.

I noticed two minor typos. I won’t turn them into a pull request as they’re so simple.

$ diff wordled-ozboware/README.md wordled
9c9
< - New score keeping function for each level/diffculty
---
> - New score keeping function for each level/difficulty
11c11
< - Home screen meny
---
> - Home screen menu



I had started playing with a radio button strategy, but I probably won’t keep working on it since you have a better approach and my Javascript skills are not that far along.

$ cat radio-button_test.js
const levels = [
    { value: 'level1', label: '<b>Level 1:</b> 100 words suitable for young children' },
    { value: 'level2', label: '<b>Level 2:</b> over 1000 commmon words' },
    { value: 'level3', label: '<b>Level 3:</b> over 2500 words' },
    { value: 'level4', label: '<b>Level 4:</b> over 3400 words' },
    { value: 'level5', label: '<b>Level 5:</b> over 13,000 words' },
];

// generate the radio groups
const group = document.querySelector('#group');
group.innerHTML = levels
    .map(
        item => `<div>
                <input type="radio" name="level" value="${item.value}" id="${item.value}">
                <label for="${item.value}">${item.label}</label>
            </div>`
    )
    .join(' ');

// add an event listener for the change event
const radioButtons = document.querySelectorAll('input[name="level"]');
for (const radioButton of radioButtons) {
    radioButton.addEventListener('change', showSelected);
}

function showSelected(e) {
    console.log(e);
    if (this.checked) {
        document.querySelector('#output').innerText = `You selected ${this.value}`;
    }
}


> On Feb 3, 2022, at 5:52 AM, ozboware ***@***.***> wrote:
> 
> 
> I've updated the game to include scores, multiple level and difficulty choice and a start up menu which includes access to high scores, best streaks, instructions and difficulty/level changes. I've also included a few animations and modals for the notifications as well as added in some responsive design. It can also be demoed in full online at wordled.online <https://wordled.online/>

Monty committed:
Merge branch 'ozboware:main' into main

Monty committed:
Toggle most screenshots

Readme with 14 screenshots was too long

Monty committed:
Switch back to simpler game

Monty committed:
Create .gitignore

Monty committed:
Use screenshot from this game

Monty committed:
Update README

Monty committed:
Fix broken link

Monty committed:
Clarify instructions

Monty committed:
Slight changes to spacing

Monty committed:
Different screenshot

Monty committed:
Add my name to copyright

Monty committed:
Switch to radio buttons to select difficulty

Logue1021 committed:
Disable radio buttons with keyboard presses

This makes it so that the radio buttons are disabled when using the keyboard to type as well

Monty committed:
Merge pull request #1 from Logue1021/patch-1

Disable radio buttons with keyboard presses

Monty committed:
Clarify comments

Monty committed:
Remove code smell

Only allow one addition to keyType in keyboard.className

Monty committed:
Fix greenLetters problem

Gold letters in blocks were green on keyboard. Now only green ones are.

Monty committed:
prettify game.js

Monty committed:
Update screenshot

Monty committed:
Fix coloring bug

Oops! blockGray should have been blockGray.

Monty committed:
Switch to on-screen keyboard for restart

Monty committed:
Simplify levels

Monty committed:
Bold selected radio button

Update style.css

Monty committed:
Shrink slightly

Monty committed:
Update README

Monty committed:
Reduce size of screenshot

Monty committed:
Set theme jekyll-theme-midnight

Monty committed:
Update README.md

Monty committed:
Update _config.yml

Monty committed:
Set theme jekyll-theme-tactile

Monty committed:
Set theme jekyll-theme-midnight

Monty committed:
Update _config.yml

Monty committed:
Set theme jekyll-theme-slate

Monty committed:
Set theme jekyll-theme-tactile

Monty committed:
Delete _config.yml

Remove theme, only leave playable game on github.io
